pynecone/app.py

```diff
@@ -85,24 +100,6 @@ def __call__(self) -> FastAPI:
         """
         return self.api
 
-    def add_default_endpoints(self):
```

Comment:
We should keep the `ping` endpoint

Reply:
I've added it back in. For future reference it can be tested like so:

```
    socket.current.emit('ping')
    console.log('ping')
    socket.current.on('ping', function(data) {
      console.log(data);
    });
```

In the console you should see:

```
ping
pong
```